src/backend/metrics/types.ts

```diff
@@ -89,6 +89,33 @@ export interface GameInstallFailed {
   sensitiveProperties?: never
 }
 
+export interface GameInstallCanceled {
+  event: 'Game Install Canceled'
+  properties: {
+    store_name: Runner
+    game_title: string
+  }
+  sensitiveProperties?: never
+}
+
+export interface GameInstallPaused {
+  event: 'Game Install Paused'
+  properties: {
+    store_name: Runner
+    game_title: string
+  }
+  sensitiveProperties?: never
+}
+
+export interface GameInstallResumed {
+  event: 'Game Install Resumed'
+  properties: {
+    store_name: Runner
+    game_title: string
+  }
+  sensitiveProperties?: never
+}
+
```

Comment:
should we keep these properties at parity with the other game install events?

```
    store_name: Runner
    game_name: string
    game_title: string
    platform: InstallPlatform | AppPlatforms
 ```

Reply:
This was my first approach but it is not possible to have reliable `platform` on GamePage and GameCard since they know nothing about the current installation besides the progress. So we would need to make them aware of that. So I decided to remove it and keep it more simple.

Are we using the appname for anything?
Also, the platform would be useful basically only if we decide to have a report like:
- Most paused games by platform.
- Most Resumed gamed by platform.
- Most canceled installation games by platform.
What do you think?

Reply:
technically the appName is the only truly unique identifier of the game. We could potentially have a game dev submit the same title for two separate game listings, which has come close to happening with The Wake for instance. 

Plus if we want to get the rest of the game's metadata, it will be simpler if we have the appName.

If we move these events to being called from the backend, I think we'd be able to send platform too. If it's too much of a hassle to get platform in that context, then sure we can ignore.

Reply:
Yeah, makes sense. I will re-add the appName.

Reply:
done